setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='zeit.content.cp',
-    version='3.23.4.dev0',
+    version='3.24.0.dev0',
```

Comment:
Hmm, das paßt aber dann nicht zum [CHANGES.txt](https://github.com/ZeitOnline/zeit.content.cp/blob/4f67ea75a398e4fd389f6d5b2beca37f21f0612d/CHANGES.txt)...

Reply:
Äh. Ja. Da war ich zu faul, weil der zest.releaser die nächste Version anhand der setup.py vorschlägt. Ich gewöhns mir ab, das ist ja unnötig verwirrend, und die eine Zeile mehr tippen kriech ich hin.

Reply:
Ach so, das wusste ich noch nicht.  Aber dann find ich das auch in Ordnung so...